kingpin/actors/rightscale/server_array.py

```diff
@@ -996,6 +1002,19 @@ def _wait_until_healthy(self, array, sleep=60):
             if min_count <= count:
                 raise gen.Return()
 
+            all_instances = yield self._client.get_server_array_current_instances(array)
+            stranded_instances = yield self._client.get_server_array_current_instances(
+                array, filters=['state==stranded'])
+
+            self.log.info('Found %d stranded instances.' % len(stranded_instances))
+
+            if (len(all_instances) - len(stranded_instances) < min_count and
```

Comment:
This condition is a bit hard to read. Could you provide some named vars here? If I'm reading it correctly the first conditional could just use `len(instances)` ?